Remotion/Web/Core/UI/Controls/DatePickerPage.cs

```diff
@@ -54,14 +54,14 @@ public class DatePickerPage : Page
   public const string CultureParameterName = "Culture";
   public const string UICultureParameterName = "UICulture";
 
-  protected HtmlHeadContents HtmlHeadContents;
-  protected Calendar Calendar;
+  protected HtmlHeadContents? HtmlHeadContents;
```

Comment:
I think we should make them not-null. everything after the OnInit, the values are initialized.